NewKernel_d/include/CGAL/NewKernel_d/functor_tags.h

```diff
@@ -201,6 +201,7 @@ namespace CGAL {
         CGAL_DECL_COMPUTE(Squared_distance);
         CGAL_DECL_COMPUTE(Squared_distance_to_origin);
         CGAL_DECL_COMPUTE(Squared_length);
+        CGAL_DECL_COMPUTE(Construct_bbox);
```

Comment:
I think it makes more sense as CGAL_DECL_MISC. "Compute" is for things that return a Lazy_exact_nt in Epeck_d.
And then Lazy_cartesian needs something similar to Point_dimension_tag, assuming that the Bbox is supposed to be built from the approximate object (maybe merge the cases Point_dimension_tag, Vector_dimension_tag, and this new one in a partial specialization `struct Functor<T,D,Misc_tag>`?)